src/module/config.js

```diff
@@ -426,6 +426,13 @@ SFRPG.skills = {
     "sur": "SFRPG.SkillSur"
 };
 
+SFRPG.controlSkills = {
+    "pil": "SFRPG.SkillPil",
+    "ath": "SFRPG.SkillAth",
+    "sur": "SFRPG.SkillSur",
+    "none": "SFRPG.None",
```

Comment:
trailing comma

Reply:
I like trailing commas. Don't you? They make latter additions and deletions easier and keeps diffs / git-blames more meaningful. When I've got multi-line objects like this, I think of commas like semicolons.

Though I've only now noticed the `.eslintrc.json` file, which is close enough to a style guide and considers it an error, so I'll change it.

By the way, eslint on develop reports 273 errors and 177 warnings.

Reply:
Are these additions directly related to the replacement of select with selectOptions?

Reply:
Ah wait I see, they're adding in values that used to be hardcoded into the sheets 👍